### PR TITLE
Implement static redis sampler

### PR DESCRIPTION
**Pull request by yannikschaelte**
Note: The dynamic sampler was not touched, apart from adding a "MODE" flag. It was just revamped in order to exploit synergies of both dynamic and static sampler.

### CODE REVIEW

File: pyabc/sampler/redis_eps/sampler.py [hunk 1]
```diff
@@ -270,37 +319,19 @@ def clear_generation_t(self, t: int) -> None:
...
+         .delete(idfy(N_ACC, ana_id, t))
+         .delete(idfy(N_REQ, ana_id, t))
+         .delete(idfy(N_FAIL, ana_id, t))
+         .delete(idfy(ALL_ACCEPTED, ana_id, t))
+         .delete(idfy(N_WORKER, ana_id, t))
+         .delete(idfy(BATCH_SIZE, ana_id, t))
+         .delete(idfy(IS_LOOK_AHEAD, ana_id, t))
+         .delete(idfy(MODE, ana_id, t))
+         .delete(idfy(QUEUE, ana_id, t))
+         .execute())
```
Comment (FelipeR888) [inline pyabc/sampler/redis_eps/sampler.py:334]: How come the only thing happening in the sample_until_n_accepted function is adding and deleting to the pipeline?

Comment (FelipeR888) [reply pyabc/sampler/redis_eps/sampler.py:334]: Nevermind, i just missed the button to open the rest ^^

Comment (FelipeR888) [reply pyabc/sampler/redis_eps/sampler.py:334]: But still, all those pipeline.set and pipeline.delete parts used to be in a separate function, didnt they?

**Review by FelipeR888 — APPROVED** [review]

File: pyabc/sampler/redis_eps/sampler.py (hunk 1 above)
Comment (yannikschaelte) [reply pyabc/sampler/redis_eps/sampler.py:334]: They are actually. GitHub just displays the difference very weirdly. They should be in start_generation_t and clear_generation_t functions if I remember correctly.

**Pull request merged**